validation/1.4/modules/ROOT/pages/validation-examples.adoc

```diff
@@ -0,0 +1,583 @@
+. Drag *Is not elapsed* to the Studio canvas.
+. In the *Is not elapsed* configuration screen, complete the following parameters:
+
+* *Time* +
+`20`
+* *Time unit* +
+`SECONDS`
+* *Since*
+`vars.time`
+
+.Is not elapsed time Configuration
+image::validation-isnotelapsed.png[.Is not elapsed time Configuration]
+
+In the XML editor, the configuration looks like this:
+
+[source,xml,linenums]
+----
+<validation:is-not-elapsed time="20" timeUnit="SECONDS" since="#[vars.time]"/>
+----
+
+
+== Configure Is Not Empty Collection Operation
+
+The *Is not empty collection* parameter validates that a value is not an empty collection. This value defaults to the payload.
+
+The following example shows how to configure the operation in Anypoint Studio:
+
+. In the *Mule Palette* view, select *Validation > Is not empty collection*.
+. Drag *Is not empty collection* to the Studio canvas.
+. In the *Is not empty collection* configuration screen, set the *Value* field to `payload`.
+
+.Is not empty collection Configuration
+image::validation-isnotemptycollection.png[Is not empty collection Configuration]
+
+In the XML editor, the configuration looks like this:
+
+
+[source,xml,linenums]
+----
+<validation:is-not-empty-collection values="#[payload]"/>
+----
+
+== Configure Is Not Null Operation
+
+The *Is not null* operation validates that the given value is not `null`.
+
+The following example shows how to configure the operation in Anypoint Studio:
+
+. In the *Mule Palette* view, select *Validation > Is not null*.
+. Drag *Is not null* to the Studio canvas.
+. In the *Is not null* configuration screen, complete the following parameters:
+
+* *Value* +
+`payload`
+* *Message* +
+`Null is not a valid value`.
+
+.Is not null Configuration
+image::validation-isnotnull.png[Is not null Configuration]
+
+In the XML editor, the configuration looks like this:
+
+[source,xml,linenums]
+----
+<validation:is-not-null value="#[payload]" message="Null is not a valid value"/>
+----
+
+== Configure Is Null Operation
+
+The *Is Null* operation validates if the provided value is `null`.
+
+The following example shows how to configure the operation in Anypoint Studio:
+
+. In the *Mule Palette* view, select *Validation > Is null*.
+. Drag *Is null* to the Studio canvas.
+. In the *Is null* configuration screen, set the *Value* field to `payload`.
+
+.Is null Configuration
+image::validation-isnull.png[Is null Configuration]
+
+In the XML editor, the configuration looks like this:
+[source,xml,linenums]
+----
+<validation:is-null value="#[payload]"/>
+----
+
+== Configure Is Number Operation
+
+The *Is number* operation verifies that a String can be parsed as a Number of a specified numeric type, using the given locale (defaulting to the System Locale) and pattern (which defaults to the Locale pattern). This validation can include a range in which the number is valid using the *Min value* and *Max Value* parameters.
```

Comment:
Use "defaulting" or "which defaults" for both parts of the sentence.
is valid > is validated